types/apm.sh

```diff
@@ -5,21 +5,27 @@ action=$1
 pkgname=$2
 shift 2
 
+if hash "apm" 2>/dev/null; then
+  bin="apm"
+elif hash "apm-beta" 2>/dev/null; then
+  bin="apm-beta"
```

Comment:
Should this be rather switched? E.g. `apm-beta` falls back to `apm`?

Reply:
Yeah, probably.